app/main.py

```diff
@@ -2,0 +1,29 @@
+import os
+
+from contextlib import contextmanager
+
+
+@contextmanager
+def move_file(command: str) -> None:
+    command_list = command.split()
```

Comment:
The `command.split()` method is called multiple times. Consider storing the result in a variable and reusing it to improve efficiency, as recommended in the checklist.